src/components/SidePanel/TraceOptions.tsx

```diff
@@ -31,27 +34,46 @@ export default () => {
         <CollapsibleGroup defaultCollapsed={false} heading="TRACE OPTIONS">
             <DatabaseFileOverride />
             <Serialports />
-            {is9160DK(device) && <TraceSettings />}
+            {is9160DK(device) && <ResetOnStart />}
+            <RefreshOnStart />
             <TraceFormatSelector />
             <TraceFileInformation />
         </CollapsibleGroup>
     );
 };
 
-const TraceSettings = () => {
+const ResetOnStart = () => {
     const dispatch = useDispatch();
     const isTracing = useSelector(getIsTracing);
     const resetDevice = useSelector(getResetDevice);
 
-    const dispatchToggle = (fn: (param: boolean) => void) => (value: boolean) =>
-        dispatch(fn(value));
-
     return (
         <Toggle
             label="Reset device on start"
             disabled={isTracing}
             isToggled={resetDevice}
-            onToggle={dispatchToggle(setResetDevice)}
+            onToggle={toggled => dispatch(setResetDevice(toggled))}
+        />
+    );
+};
+
+const RefreshOnStart = () => {
+    const dispatch = useDispatch();
+    const isTracing = useSelector(getIsTracing);
+    const refreshOnStart = useSelector(getRefreshOnStart);
+    const selectedFormats = useSelector(getTraceFormats);
+    const liveEnabled = selectedFormats.includes('live');
+    const title = liveEnabled
+        ? 'Cannot be enabled when Wireshark is enabled.'
+        : 'Refresh dashboard 5 seconds after starting the trace';
```

Comment:
The specifications says to have `Cannot be enabled when Wireshark is enabled.`, even when Wireshark is not enabled. I think it's for consistency, but I do also like that detailed description. Could this be merged as one text? Something like this:

```typescript
const title = 'Cannot be enabled when Wireshark is enabled.\nRefresh dashboard 5 seconds after starting the trace';
```